python/revng/cli/_commands/daemon.py

```diff
@@ -68,6 +68,8 @@ def run(self, options: Options):
             elif bind.startswith("unix:"):
                 _, path = bind.split(":", 1)
                 args.extend(["--uds", path])
+            elif bind == "none":
+                pass
```

Comment:
Can you doc this in the help of the `-b` argument?

Reply:
Done